### DISCUSSION TITLE
Cannot update my fork

### DISCUSSION THREAD

**Discussion by epernod** [jnbrunet/caribou#52] [Q&A] [answered]
Hi, 
I have a problem updating my fork with recent commit. Error message: 
```
To https://github.com/epernod/caribou.git
 ! [remote rejected] master -> master (refusing to allow an OAuth App to create or update workflow `.github/workflows/macos.yml` without `workflow` scope)
error: failed to push some refs to 'https://github.com/epernod/caribou.git'
```

**Comment by epernod** [comment, marked as answer]
sorry was not related to recent commit. (you can delete this discussion)
https://stackoverflow.com/questions/64059610/how-to-resolve-refusing-to-allow-an-oauth-app-to-create-or-update-workflow-on

**Reply by jnbrunet** [reply]
I'll keep it since it can happen to others. Thanks for linking the solution !
